judgels-backends/sandalphon/sandalphon-api/src/main/java/judgels/sandalphon/api/submission/Grading.java

```diff
@@ -9,6 +9,11 @@
 @Value.Immutable
 @JsonDeserialize(as = ImmutableGrading.class)
 public interface Grading {
+    Verdict PENDING = Verdict.of("?", "Pending");
```

Comment:
No, I mean we can just use e.g. `Verdicts.PENDING`, no need to create new vars

Reply:
Oh my bad